functions/.gitignore

```diff
@@ -1,6 +1,6 @@
 # Compiled JavaScript files
-**/*.js
-**/*.js.map
+lib/*.js
+lib/*.js.map
```

Comment:
@firebase 

Firebase CLI documentation about `firebase init` is very clear. However I found a bug in the created project and have no clue where to report it (i checked firebase github but there are too many projects that all committed in the last 5 days).

I'll report the bug here.

firebase init with firestore,functions,hosting,storage,emulators enabled and everything at the default with typescript and eslint set to true, generated a functions folder that has a gitignore that ignores **/*.js instead of lib, causing .eslintrc.js not to be committed, causing CI to fail deployment because eslint config can not be found.